etc/kayobe/ansible/templates/os_capacity-clouds.yml.j2

```diff
@@ -1,10 +1,13 @@
 clouds:
   openstack:
     auth:
-      auth_url: "{{ secrets_os_exporter_auth_url }}"
-      application_credential_id: "{{ secrets_os_exporter_credential_id }}"
-      application_credential_secret: "{{ secrets_os_exporter_credential_secret }}"
+      auth_url: "{{ secrets_os_capacity_auth_url }}"
+      application_credential_id: "{{ secrets_os_capacity_credential_id }}"
+      application_credential_secret: "{{ secrets_os_capacity_credential_secret }}"
     region_name: "RegionOne"
     interface: "internal"
     identity_api_version: 3
     auth_type: "v3applicationcredential"
+{% if kolla_enable_tls_internal | bool %}
+    verify: False
+{% endif %}
```

Comment:
It should be possible to specify cacert instead, in the same way as is done in ansible/roles/prometheus/templates/clouds.yml.j2

Reply:
Please could you comment on how this was resolved @assumptionsandg? Thanks!

Reply:
I'm happy for this to be followed up in  another PR

Reply:
@assumptionsandg Is this going to be done in another PR or this one?

Reply:
Probably a different pull request. This causes some issues as I don't think the keystone certificates exist on monitoring nodes.

Reply:
Implemented in #896